pkg/kubelet/cm/devicemanager/topology.go

```diff
@@ -72,11 +73,9 @@ func (m *ManagerImpl) GetTopologyHints(pod v1.Pod, container v1.Container) ([]to
 				allDeviceSocketsInt = append(allDeviceSocketsInt, socket)
 			}
 			crossSocketMask, _ := socketmask.NewSocketMask(allDeviceSocketsInt...)
-			klog.Infof("CrossSocketMask: %v", crossSocketMask.String())
-			deviceHints = append(deviceHints, topologymanager.TopologyHint{SocketMask: crossSocketMask})
+			deviceHints = append(deviceHints, topologymanager.TopologyHint{SocketAffinity: crossSocketMask, Preferred: false})
 		}
-		admit = calculateIfDeviceHasSocketAffinity(deviceHints)
-		klog.Infof("DeviceHints: %v, Admit: %v", deviceHints, admit)
+		klog.Infof("[devicemanager-topology] DeviceHints: %v", deviceHints)
 	}
 
 	return deviceHints, admit
```

Comment:
I think you're returning two values here but the new function signature at line 13 only specifies one return value.